admin_guide/manage_nodes.adoc

```diff
@@ -537,6 +537,8 @@ node-config-master-infra   1         1d
 ----
 
 .Sample configuration map for the *node-config-compute* group
+====
```

Comment:
@mhutter We have been removing the `====` around code examples for some time. Adding these here would be the only place in the file that has them. Unless you have a specific reason for adding `====` that I am not aware of.

Reply:
Ah, no I have not :)